.github/workflows/dapr_cli.yml

```diff
@@ -17,8 +17,19 @@ on:
       - master
       - release-*
 jobs:
+  pre_job:
+    name: Skip Duplicate Actions
+    runs-on: ubuntu-latest
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@v3.4.0
+        with:
+          paths_ignore: '["**.md", ".codecov.yaml", ".github/**/**.yml", "CODEOWNERS"]'
```

Comment:
Why `.github/**/**.yml`? We absolutely want to run the tests when we make changes to the github actions workflows. How else will we iterate on them?